server/routes/auth.routes.js

```diff
@@ -662,41 +705,56 @@ module.exports = {
    * @param req
    * @param res
    */
-  casStage2 : async function (req, res) {
-    /** @namespace req.session.cas_userinfo */
-
-    if ( ! ( req.session && req.session['cas_userinfo'] && (req.session.cas_userinfo['max-id'] || req.session.cas_userinfo['maxId']  ))) {
-      // didn't get CAS session info
+  casStage2: async function(req, res) {
```

Comment:
Not sure why you updated this since, this only affects Max.gov login which is not being utilized anymore. Was there a reason?